openedx/core/djangoapps/course_live/serializers.py

```diff
@@ -52,10 +52,10 @@ def create(self, validated_data):
             if key in set(self.Meta.fields).difference(self.Meta.read_only):
                 setattr(instance, key, value)
 
-        pii_sharing_allowed = self.context.get('pii_sharing_allowed', False)
+        share_email, share_username = self.pii_sharing_allowed()
```

Comment:
If pii sharing is not allowed/required it would be better to not send these variables at all rather than sending an empty or false value.